docs/layouts/shortcodes/generated/table_config_configuration.html

```diff
@@ -22,7 +22,7 @@
         </tr>
         <tr>
             <td><h5>table.generated-code.max-length</h5><br> <span class="label label-primary">Batch</span> <span class="label label-primary">Streaming</span></td>
-            <td style="word-wrap: break-word;">64000</td>
+            <td style="word-wrap: break-word;">4000</td>
             <td>Integer</td>
             <td>Specifies a threshold where generated code will be split into sub-function calls. Java has a maximum method length of 64 KB. This setting allows for finer granularity if necessary.</td>
```

Comment:
You need update `Java has a maximum method length of 64 KB` too. Why not 64K?
Add some description for `JIT` limitation?